### DISCUSSION TITLE
I can't upload files.

### DISCUSSION THREAD

**Discussion by BKSVitor** [abraunegg/onedrive#2705] [General]
Hello everybody.

In the last few days I noticed that my files are not uploading. Download works great but Upload doesn't seem to work.

I'm not sure what it could be, as I didn't find any log indicating an upload error, maybe it's due to the amount of files I download daily?

To summarize my work, I exchange files daily with approximately 1,000 files, where I have a Windows server synchronizing documents and an Ubuntu server (backup) which I use as a repository. Files synced on Windows are downloading to Ubuntu, but files I create on Ubuntu are not uploading to Windows.

![Imagem2024-04-26_ 09 36 55_6d692e25](https://github.com/abraunegg/onedrive/assets/142534915/e2fe1189-b605-4f4b-8787-12017c392f18)
![Imagem2024-04-26_ 09 34 59_ed321120](https://github.com/abraunegg/onedrive/assets/142534915/75955184-f7bd-4582-b28b-f1142b00974e)

The files and directories highlighted in yellow were used for upload tests and as we can see they were not synchronized.

These files were added to the OneDrive directory on Monday.

Is there any parameter or command that uploads my files and does not stop the download? Full_scan is enabled in my configuration file.

**Comment by abraunegg** [comment]
@BKSVitor 
> In the last few days I noticed that my files are not uploading. Download works great but Upload doesn't seem to work.

This is very odd. By default the application will upload|download files correctly. You can then force the client to perform one-way sync operations via:
* `--download-only` - this will *only* download data
* `--upload-only` - this will *only* upload data

A few questions:
* What application version are you using?
* What application command(s) are you using when running the application?
* Are you expecting the files to be automatically uploaded when added? or are you only wanting the files to be uploaded when you run a specific command?
* What is the application output saying? Create a logfile by enabling logging and or provide console output - start with: `onedrive --synchronize --verbose > debug_output.log 2>&1` to generate an applicable verbose log.
* Does the application, when running as your normal user have access to 'read' the files? In your screenshot, you are showing that you need to use `sudo` to access your `/media/casmed/onedrive` folder ... the client should not be run using sudo (as you end up running the client then as root, not the user it should be run as) thus, it might be looking for files in the wrong location.
* Are you trying to run the client using 'sudo' ?

Outside of this, please read the documentation.
